org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="PowerStatus" type="s" access="readwrite">
+      <description>Get/Set power status. (On, Off)</description>
+    </property>
+
```

Comment:
Should it be boolean?

Also, is this really separate from the Mode property? While I don't have experience with Wink, other smart thermostat APIs have included "Off" as one of the available modes.